api/v2/api.go

```diff
@@ -130,11 +130,24 @@ func NewAPI(
 	openAPI.SilencePostSilencesHandler = silence_ops.PostSilencesHandlerFunc(api.postSilencesHandler)
 
 	handleCORS := cors.Default().Handler
-	api.Handler = handleCORS(openAPI.Serve(nil))
+	api.Handler = handleCORS(setResponseHeaders(openAPI.Serve(nil)))
 
 	return &api, nil
 }
 
+var responseHeaders = map[string]string{
+	"Cache-Control": "no-cache, no-store, must-revalidate",
```

Comment:
The specific headers I chose exactly match what V1 sends. Though, perhaps now is the time to revisit this?

Specifically, `no-store` is stronger than the other two headers and disables caching completely **in later specifications.** In older specifications, it made sense to use both `no-cache` and `no-store` at the same time. So, I kept both. See: https://stackoverflow.com/questions/866822/why-both-no-cache-and-no-store-should-be-used-in-http-response

Reply:
I'd say we drop it and keep only `no-store` the specification of `no-store` is quite old so it seems sensible to rely on clients having implemented that already.

Reply:
Done!